recipe_scrapers/_abstract.py

```diff
@@ -79,19 +87,19 @@ def category(self):
         raise NotImplementedError("This should be implemented.")
 
     def total_time(self):
-        """total time it takes to preparate and cook the recipe in minutes"""
+        """total time returns the time needed to prepare and cook the recipe in minutes"""
         raise NotImplementedError("This should be implemented.")
 
     def cook_time(self):
-        """cook time of the recipe in minutes"""
+        """cook time returns the time needed to cook the recipe in minutes"""
         raise NotImplementedError("This should be implemented.")
 
     def prep_time(self):
-        """preparation time of the recipe in minutes"""
+        """prep time returns the time needed to prepare the recipe in minutes"""
         raise NotImplementedError("This should be implemented.")
 
     def yields(self):
-        """The number of servings or items in the recipe"""
+        """yields returns the total servings or items in the recipe"""
```

Comment:
If we're going to modify these, let's:

* Use shorter phrases, like the existing descriptions (people using the library are likely to know, or find out quickly, that these are method calls not properties).
* Complete the docstrings for the all the key recipe fields (`author`, `title`, `ingredients`, `instructions`).
* Review by running a Python interpreter and running:

```pyi
>>> from recipe_scrapers import AbstractScraper
>>> help(AbstractScraper)
```

We want the output to be concise, quick to scan, and descriptive about the purpose of each field.

Reply:
Took a shot at applying these to all the fields, let me know if this is what you had in mind?

Reply:
Almost exactly, thank you!  The repetition of `Returns ...` doesn't seem perfect though.  Thinking about it..

Reply:
Would definitely still work if `Returns` was just removed leaving `Title of recipe`, `Category of recipe` 

Or I think using returns could work here if it was swapped around to be `Returns recipe's title`,`Returns recipe's category`

Reply:
> Would definitely still work if `Returns` was just removed leaving `Title of recipe`, `Category of recipe`

That sounds good I think.  Let's try that and see how the results turn out?

Reply:
Just pushed the change, I think this looks like a good fit.

Reply:
Great!  Yep, that looks much better to me too.  Thank you!

Reply:
Awesome, I think we're ready for merge here. Any final changes or want me to hold off?